From 03abc329e1bc5b54945113f55bddf3ef95974bb2 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Fri, 30 Aug 2013 08:51:22 +0200
Subject: [PATCH 03/29] Remove usb wacom tablet

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <b28882d567fa02f578504ff44ec577bbafc68c29.1377851454.git.mrezanin@redhat.com>
Patchwork-id: 53935
O-Subject: [RHEL7 qemu-kvm PATCHv2 2/6] Remove usb wacom tablet
Bugzilla: 903914
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Michal Novotny <minovotn@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

BZ: 903914
Upstream: n/a, RHEL only

We do not support wacom tablet, disabling it.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/usb.mak | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/usb.mak |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/default-configs/usb.mak b/default-configs/usb.mak
index 9a3bd09..39a3552 100644
--- a/default-configs/usb.mak
+++ b/default-configs/usb.mak
@@ -1,4 +1,3 @@
-CONFIG_USB_TABLET_WACOM=y
 CONFIG_USB_STORAGE_BOT=y
 CONFIG_USB_STORAGE_UAS=y
 CONFIG_USB_SMARTCARD=y
-- 
1.7.1

